client/src/components/Footer/Footer.css

```diff
@@ -0,0 +1,56 @@
+footer {
+    background-color: hsl(0, 0%, 70%);
```

Comment:
we can use the global variables now
```suggestion
    background-color: var(--gray70);
```

Reply:
(don't commit this suggestion using the button above, just change it in your own code)

Reply:
Great, all done.